website/src/pages/pricing.tsx

```diff
@@ -82,7 +82,7 @@ export default ((props: any) => (
                             <PricingPlan
                                 className="pricing-page__plan"
                                 name="Team"
-                                price={<div className="text-center">Starts at $150/mo</div>}
+                                price={<div className="text-center">$150/mo</div>}
```

Comment:
@aileenrose is this the price per user?

Reply:
$150/mo for up to 25 users.  what's the best way to phrase it?

Reply:
moving the conversation out of slack here - @abschu suggested changing both free an team tiers to "includes # users"